src/main/java/com/skhu/luxuryshop/controllerAdvice/ProductExceptionControllerAdvice.java

```diff
@@ -0,0 +1,31 @@
+package com.skhu.luxuryshop.controllerAdvice;
+
+
+import com.skhu.luxuryshop.exception.ProductFindByIdException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+@Slf4j
+public class ProductExceptionControllerAdvice {
+
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    public ResponseEntity<String> methodArgumentNotValidException(MethodArgumentNotValidException e) {
+        log.error( e.getMessage(), e );
+        return new ResponseEntity( e.getMessage(), HttpStatus.BAD_REQUEST );
+    }
```

Comment:
```suggestion
    @ExceptionHandler(MethodArgumentNotValidException.class)
    public ResponseEntity<String> methodArgumentNotValidException(MethodArgumentNotValidException e) {
        log.error( e.getMessage(), e );
        return new ResponseEntity( e.getMessage(), HttpStatus.BAD_REQUEST );
    }
```

ResponseEntity에 HttpStatus를 인자로 보내고있으니 `@ResponseStatus`는 필요 없어 보입니다 :)